### DISCUSSION TITLE
Add display flag Icon option

### DISCUSSION THREAD

**Discussion by SinglePole** [ppy/osu#20154] [Ideas] [answered]
For some political reasons, some regional flags cannot be displayed in Chinese Mainland. Please consider adding the option of whether to display the flag Icon on the song selection interface (excluding the leaderboard)
![osu_2022-09-06_10-57-41](https://user-images.githubusercontent.com/88593454/188539281-8ac999c0-fd94-45b0-a1f8-a1c8c07a10fc.png)

**Comment by peppy** [comment, marked as answer]
This won't be an option, sorry. If you feel strongly about it you can compile a version with the flag resources removed, but it will not be officially supported.

**Reply by SinglePole** [reply]
I'm so sorry. This is a political problem in our country. If so, in the future, when playing with the official version, Lazer will not be able to be played LIVE in Chinese Mainland (in the case of logging in to the account), because it has inappropriate content to display.

**Reply by Loreos7** [reply]
> I'm so sorry. This is a political problem in our country. If so, in the future, when playing with the official version, Lazer will not be able to be played LIVE in Chinese Mainland (in the case of logging in to the account), because it has inappropriate content to display.

You can actually just choose "local" or "details" in leaderboard so no one would see other flags
